app/src/main/java/com/practicum/playlistmaker/SearchActivity.kt

```diff
@@ -202,6 +221,7 @@ class SearchActivity : AppCompatActivity() {
                 noResultsFrame.visibility = View.GONE
                 connectTrouble.visibility = View.VISIBLE
                 search_history_ll.visibility = View.GONE
+                progressBar.visibility = View.GONE
```

Comment:
Есть удобные экстеншены:
```suggestion
                progressBar.isVisible = false
```